lib/puppetx/simp/compliance_map.rb

```diff
@@ -489,4 +489,4 @@ def lookup_fact(fact)
 
 def module_list
   @context.environment.modules.map { |obj| { "name" => obj.metadata["name"], "version" => obj.metadata["version"] } }
```

Comment:
Line 250: This `load` has not been passed the options with :aux_dirs,  :data_dir, and 'mode' keys.  The first two keys allow the user to load compliance YAML files from a different directory.  So, this means unless the enforcement hiera backend was used for looking up values first, the potentially wrong data would be used to generate reports.

Reply:
This is mostly for testing (for now). Removed from the README until we can get it fully tested under the hood.